src/components/Text.js

```diff
@@ -14,31 +14,21 @@ type Props = {
   markdown?: boolean,
   style?: StyleObj,
   allowFontScaling?: boolean,
-  onLayout?: Function
+  onLayout?: Function,
+  numberOfLines?: number
```

Comment:
Now that extra props are encapsulated within `otherProps` do we still need these prop types?

Is it because flow complains that we're passing these props down and wants to know about them?

Reply:
yes, tried to remove them and flow complains

Reply:
You might be able to do something by extending the RnText Prop type instead.

Reply:
I think intersection type is what you want: https://flow.org/en/docs/types/intersections/

Reply:
I thought about this, too. Has the issue, that `Markdown` does not support all the same props. But it might still be fine. The only requirement is, that we can actually import the default text prop types. I noticed this they are not available for all components.

Reply:
🎺